### ISSUE TITLE
Performance on Humanoid-v2?

### ISSUE THREAD

**Issue by twni2016** [sfujim/TD3#19]
Hi,

Thanks for your elegant codes. Here I'm asking to confirm how TD3 performs in mujoco Humanoid-v2. 

In SAC [paper](https://arxiv.org/pdf/1812.05905.pdf) and also other papers, they reported TD3 almost cannot make any progress on Humanoid-v2 (the learning curve is like y=0). However, we tried to reproduce this phenomenon using your code and found that TD3 could reach 6000+ scores in testing returns in 10M steps (just one seed though).
<img src="https://user-images.githubusercontent.com/20842215/72132654-7b28ef00-334d-11ea-9811-73f403ea9a1f.png" width="300">

Since you didn't report humanoid results in your paper, and you might change hyperparameters to improve the results, I'm wondering which result might be true? We use the latest hyperparams and set `start_timesteps=10000` in reference to https://github.com/sfujim/TD3/blob/master/run_experiments.sh#L11

Thanks!

**Comment by sfujim** [comment]
Hi! 

Originally, we never tested on Humanoid due to computational costs since the only paper that solved Humanoid at the time was PPO, which took ~50million time steps. Unfortunately, in the original repo there was a bug for non-integer max actions (Humanoid uses a range of [-0.4,0.4] rather than [-1,1] for whatever reason), causing TD3's actor to only output 0. This is the result reported in several papers, including SAC. 

That being said, TD3 still performed poorly with the original hyper-parameters. I have since updated TD3 to match the hyper-parameters of SAC and now it performs quite well.

Hope that answers your questions.

**Comment by twni2016** [comment]
Thanks for your response! That resolves my concerns.
